jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/ObjectRule.java

```diff
@@ -325,6 +327,7 @@ private void addHashCode(JDefinedClass jclass) {
         }
         
         for (JFieldVar fieldVar : fields.values()) {
+            if( (fieldVar.mods().getValue() & JMod.STATIC) == JMod.STATIC) continue;
```

Comment:
This change, and the others related to filtering out static fields, should probably come in as a separate PR, since this is technically broken on the 0.4.X line of development.